internal/bzlmod/go_deps.bzl

```diff
@@ -70,7 +70,7 @@ def _fail_on_non_root_overrides(module, tag_class, attribute = None):
             fail(_report_forbidden_override(module, tag_class))
 
 def _check_directive(directive):
-    if directive.startswith("gazelle:") and " " in directive:
+    if directive.startswith("gazelle:") and " " in directive and not directive[len("gazelle:"):][0].isspace():
```

Comment:
Gazelle does not mandate that there are any space characters in a directive:

https://github.com/bazelbuild/bazel-gazelle/blob/71fa620b33b23e104739a9666ad4bd11492d6d73/rule/directives.go#L75

That `\s` can be satisfied by a horizontal tab character. If we're going this far to accommodate the syntax, instead of checking if `" " in directive`, perhaps we could check for at least one character satisfying `isspace`.

I know it's uglier, so feel free to tell me that we shouldn't try that hard. That's for when Gazelle reads these directives from source files, so it makes sense that it's flexible like this. For our case, ideally we'd accept these as structs with a directive key and value as fields.

Reply:
We shouldn't try that hard 😉

I'm not even sure whether you can have tabs in starlark strings without escaping and even if you can, this is a case where we may very well want to force more normalization than we would for comments.